cogito/gchatsink.go

```diff
@@ -36,9 +36,9 @@ func (sink GoogleChatSink) Send() error {
 	}
 
 	state := sink.Request.Params.State
-	if !shouldSendToChat(state, sink.Request.Source.NotifyOnStates) {
+	if !shouldSendToChat(sink.Request) {
 		sink.Log.Debug("not sending to chat",
-			"reason", "state not in enabled states", "state", state)
+			"reason", "state not in configured states", "state", state)
```

Comment:
I'm confused here. It seems that `shouldSendToChat` now verifies also the presence of a chat content, not only the state.

Reply:
Yes, it is explained in the PR description and in the README changes in this PR.

Reply:
Yes it is. Now you could have 2 reasons to skip the chat.

Reply:
This is what I though at the beginning also, and did another implementation of `shouldSendToChat` that itself returned the reason (since it knows everything).

At the end, I decided it was not worth it, at least for the time being, because this reason "state not in configured states" is always true. It could say instead "state not in configured states and both chat_message and chat_message_file are empty", but to me it would just add confusion...

Reply:
I understand, so let keep it as is.